build/build-config.yml

```
@@ -272,9 +272,9 @@ config:
     build:
       - work-dir: "backend/individual"
         dockerfile: "build/maven-java17/Dockerfile"
-        image-name: "individual"
+        image-name: "mukta-individual"
       - work-dir: "backend/individual/src/main/resources/db"
-        image-name: "individual-db"   
+        image-name: "mukta-individual-db"   
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)